Directory.Build.props

```diff
@@ -6,11 +6,8 @@
 <Project InitialTargets="DirectoryBuildPropsInfo" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup Condition="$(BuildingWithBuildExe) != 'true'">
     <IsTDPConfiguration>false</IsTDPConfiguration>
-    <IsTDPConfiguration Condition="'$(Configuration)' == 'Debug_test'">true</IsTDPConfiguration>
```

Comment:
><IsTDPConfiguration Condition="'$(Configuration)' == 'Debug_test'">true</IsTDPConfiguration> [](http://example.com/codeflow?start=4&length=92)

now we have unrelated changes

Reply:
rebased to release/reunion-0.8

Reply:
maybe better just start fresh

Reply:
I'd rather not. It takes ~40 minutes to turn a build, plus however long we're queued. This change has no impact on the resulting Transport Package, nor even the build pipeline or inner-loop as we don't use Debug_test configs (hence the reason they've been stripped from main)

Reply:
i doubt we want to include these changes to release branch.